packages/core/src/zone/ng_zone.ts

```diff
@@ -242,6 +249,7 @@ interface NgZonePrivate extends NgZone {
   lastRequestAnimationFrameId: number;
   isStable: boolean;
   shouldCoalesceEventChangeDetection: boolean;
+  shouldCoalesceChangeDetection: boolean;
   nativeRequestAnimationFrame: (callback: FrameRequestCallback) => number;
 
   // Cache of  "fake" top eventTask. This is done so that we don't need to schedule a new task every
```

Comment:
Though the second line isn't in the diff, here's how I'd recommend the next line's edits so it makes sense together.

```suggestion
  // Cache a  "fake" top eventTask so you don't need to schedule a new task every
  //  time you run a `checkStable`.
```

Reply:
Got it, thanks.